servers/rendering_server.h

```diff
@@ -109,6 +109,7 @@ class RenderingServer : public Object {
 	virtual RID texture_proxy_create(RID p_base) = 0;
 
 	virtual void texture_2d_update(RID p_texture, const Ref<Image> &p_image, int p_layer = 0) = 0;
+	virtual void texture_2d_update_partial(RID p_texture, const Ref<Image> &p_data, Vector2i p_dst_pos, int p_dst_mipmap = 0, int p_layer = 0) = 0;
```

Comment:
Just linking back to the previous PR's reviews https://github.com/godotengine/godot/pull/75892#discussion_r1161872551

It seems like you have lost some of the API changes suggested by reviewers.